src/state/type/actions.js

```diff
@@ -0,0 +1,10 @@
+import { TYPE_SELECTED } from "../action-types";
+
+export const setType = value => (
+  dispatch => {
+    dispatch({
+      type: TYPE_SELECTED,
+      data: value
+    });
+  }
+);
```

Comment:
😶

Reply:
sorrrry!!!

Reply:
LOOOOL, missing newlines can't get past Rehan's radar :eyes:

Reply:
as an aside... prettier shouldn't even allow this to happen. not sure why prettier isn't complaining about this. but that might be worth investigating to make sure prettier is actually working as expected.